roles/servicetelemetry/defaults/main.yml

```diff
@@ -72,6 +72,9 @@ servicetelemetry_defaults:
           - collector_type: ceilometer
             subscription_address: anycast/ceilometer/metering.sample
             debug_enabled: false
+          - collector_type: sensubility
+            subscription_address: sensubility/metrics
```

Comment:
I haven't fully decided what this address should be yet, any suggestions?

Reply:
Name seems fine to me.

Reply:
Actually I think I'd like this to align to something we already have a bit. I'd suggest `sensubility/telemetry` to align to `collectd/telemetry`.